### DISCUSSION TITLE
有没有大佬知道官网“在线测试”用的是什么组件啊！！！

### DISCUSSION THREAD

**Discussion by LuJinSehng** [layui/layui#1723] [问答]
如题。官网：https://layui.dev/playground/2/  请问这个是什么组件，有文档或demo吗？
跪求，感激不尽啊！
最近有个需求，订单功能季度扎帐，需要html快照，且支持html在线编辑调整，好难啊。

**Comment by sentsim** [comment]
用的是 Monaco Editor
https://microsoft.github.io/monaco-editor
